packages/tools/src/utilities/contours/interpolation/interpolate.ts

```diff
@@ -108,7 +108,7 @@ function startInterpolation(viewportData: InterpolationViewportData) {
 
   if (interpolationList.length) {
     triggerEvent(
-      viewportData.viewport.element,
+      eventTarget,
```

Comment:
I think `eventTarget` is more appropriate here, no?

Reply:
i think this will be a breaking change, put the event Target in addition to element then in 5.0 we will remove the element sub. Also document it in https://github.com/cornerstonejs/cornerstone3D/issues/1526